### PR TITLE
update in authors.rst

### PR DESCRIPTION
**Pull request by yashu-seth**

### CODE REVIEW

**Comment by khalibartan** [comment]
Ping @ankurankan

File: AUTHORS.rst [hunk 1]
```diff
@@ -1,25 +1,64 @@
...
+
+* nehasoni <neha.soni.ece10@itbhu.ac.in>
+
+* Jaidev Deshpande <deshpande.jaidev@gmail.com>
+
+* Abhijeet Kislay <abhijeetkislay@gmail.com>
+
+* palashahuja <abhor902@gmail.com>
+
+* The Gitter Badger <badger@gitter.im>
```
Comment (khalibartan) [inline AUTHORS.rst]: @yashu-seth  remove this

Comment (yashu-seth) [inline AUTHORS.rst]: @khalibartan okay :D

File: AUTHORS.rst [hunk 2]
```diff
@@ -1,25 +1,64 @@
...
 
-* Navin Chandak
+* William Lyon <wlyon@quantpost.com>
 
-* Jaspreet
+* Navin Chandak <navinchandak92@gmail.com>
+
+* Jaspreet Singh <jaspreetsingh112@gmail.com>
+
+* nehasoni <neha.soni.ece10@itbhu.ac.in>
```
Comment (khalibartan) [inline AUTHORS.rst:28]: Neha Soni

File: AUTHORS.rst [hunk 3]
```diff
@@ -1,25 +1,64 @@
...
+
+* Jaidev Deshpande <deshpande.jaidev@gmail.com>
+
+* Abhijeet Kislay <abhijeetkislay@gmail.com>
+
+* palashahuja <abhor902@gmail.com>
```
Comment (khalibartan) [inline AUTHORS.rst:34]: Palash Ashuja

File: AUTHORS.rst [hunk 4]
```diff
@@ -1,25 +1,64 @@
...
+
+* Ashwini Chaudhary <monty.sinngh@gmail.com>
+
+* Sitesh Ranjan <siteshjaiswal@gmail.com>
+
+* Pratyaksh Sharma <pratyaksh@me.com>
+
+* Aleksandar Dimitriev <dimitriev_a@yahoo.com>
+
+* vivek425ster <vivek425ster@gmail.com>
```
Comment (khalibartan) [inline AUTHORS.rst]: Lets just keep it just Vivek

Comment (yashu-seth) [inline AUTHORS.rst]: @khalibartan I checked for their names but they have not mentioned it on their github profiles. That is why I did not mention their names in the first place. Even if the some of the names are obvious shouldn't we just give their usernames?

File: AUTHORS.rst [hunk 5]
```diff
@@ -1,25 +1,64 @@
...
+
+* palashahuja <abhor902@gmail.com>
+
+* The Gitter Badger <badger@gitter.im>
+
+* Saket Choudhary <saketkc@gmail.com>
+
+* all3fox <all3fox@gmail.com>
+
+* Vivek Jain <vivek425ster@gmail.com>
```
Comment (khalibartan) [inline AUTHORS.rst:40]: This is the real name, remove the other vivek425ster, see both names are associated with same email

**Comment by yashu-seth** [comment]
#583

**Pull request merged**